benchmark_runner/common/oc/oc.py

```diff
@@ -310,10 +319,10 @@ def get_vm(self, label: str = '', namespace: str = environment_variables.environ
         """
         if label:
             return self.run(
-                cmd="oc get vmi -n " + namespace + " --no-headers | awk '{ print $1; }' | grep -w " + label,
+                cmd=f"o{self.__cli}c get vmi -n " + namespace + " --no-headers | awk '{ print $1; }' | grep -w " + label,
```

Comment:
Same: do it as a single f-string.

Reply:
I changed it to f string, what is the purpose of variable `cmd` here?

Reply:
It's treating it as a keyword although it's a positional parameter.  It doesn't do any harm, but it shouldn't really be needed here.